web/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,4 @@
 export const environment = {
-  production: true
+  production: true,
```

Comment:
idk about that

Reply:
It is default property, may be useful in some cases

Reply:
It is being used in main.ts file